src/screens/AddScreen.js

```diff
@@ -100,7 +128,7 @@ export default function AddScreen({ navigation }) {
           lat =>
             setEntry({
               ...entry,
-              location: { ...entry.location, latitude: Number(lat) },
+              location: { ...entry.location, latitude: lat },
```

Comment:
Create an error where if the "lat" they input does not convert to a Number (ex: they input alphabet letters), show an error to the user when they try to submit the Tree with the invalid latitude.

Do the same for "long".